src/routes/styleguide/components/ColorBox.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
```

Comment:
This kind of relates to what we were talking about yesterday: We now have an extra component that we're not _actually_ using. Over time this is just going to add a bit of friction and confusion. I can see something like this in something like a storybook (I'm using lowercase here because I don't care if it's _actually_ Storybook, but just the general idea), but it doesn't belong in the actual codebase.